src/client/components/Forms/SignUp.js

```diff
@@ -1,54 +1,181 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import iconGithub from '../../assets/images/icons/github.svg';
+import './SignUp.style.css';
+import Button from '../Button/Button';
 
 export default function SignUp({ onSubmit }) {
+  const [name, setName] = useState('');
   const [email, setEmail] = useState('');
+  const [position, setPosition] = useState('');
+  const [image, setImage] = useState('');
+  const [linkedin, setLinkedin] = useState('');
+  const [github, setGithub] = useState('');
   const [password, setPassword] = useState('');
   const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [validation, setValidation] = useState('');
+
   const handleEmailInput = (e) => setEmail(e.target.value);
+  const handleNameInput = (e) => setName(e.target.value);
+  const handlePositionInput = (e) => setPosition(e.target.value);
+  const handleImageInput = (e) => setImage(e.target.value);
+  const handleLinkedinInput = (e) => setLinkedin(e.target.value);
+  const handleGithubInput = (e) => setGithub(e.target.value);
   const handlePasswordInput = (e) => setPassword(e.target.value);
   const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
   const handleSubmit = (e) => {
     e.preventDefault();
-    onSubmit({ email, password, passwordConfirm });
+    if (password !== passwordConfirm) {
+      return setValidation('**Your password does not match**');
+    }
+    setValidation('');
+    onSubmit({
+      email,
+      password,
+      passwordConfirm,
+      name,
+      position,
+      image,
+      linkedin,
+      github,
```

Comment:
```suggestion
     email,
    password,
    passwordConfirm,
    fullName,
    position,
    linkedin,
    github,
    website,
```

Reply:
Kamal, i made a suggestion to you since I saw you didn´t change the props according to my file.. i guess maybe it is best we ask the project lead what to do about this?